src/main/java/ch/admin/bag/covidcertificate/sdk/core/extensions/TestEntryExtensions.kt

```diff
@@ -68,3 +69,7 @@ fun TestEntry.validFromDate(): LocalDateTime? {
 	return DateUtil.parseDateTime(this.timestampSample)
 }
 
+fun  TestEntry.isSeroPositiv(): Boolean {
```

Comment:
confusing naming, should this not also check the result of the test?

Reply:
No, this function is only responsible for displaying content and show a Sero Postiv Recovery instead of a Test.
But for the naming probably : isSeroPositivTest()